### ISSUE TITLE
[Feature request] Merge graphs

### ISSUE THREAD

**Issue by frapik99** [axt/angr-utils#13]
Hi @axt,
i was thinking that would be nice to have the opportunity to merge 2 graphs. When this can be useful? Suppose that a user would prefer to generate a graph of a function in a shared object and generate the graph of function contained in an elf file that uses that shared object. At this point the user could just generate the graph from the shared object only once and after for each occurrence pf that specific function in the elf it could just "merge" the pre-generated graph. Would be this useful?

Thanks

**Comment by axt** [comment]
Hi @frapik99. 

First of all, sorry for my late answer. 

To be honest I don't like the idea of adding this kind of complexity/functionality to bingraphvis! 
I would propose you to do this kind of merge of the CFG-s outside of the framework, and then call bingraphvis with the merged CFG to plot it. Anyhow, If you can create a working example of this, I would consider to add it to the framework, if I can find a place where I can fit it.

Also: I noticed you closed the other ticket. Did we solve all the issues there, or you closed it because I wasn't responding. If you have issues left, I have a few days now to check them, just let me know.
